pkg/datarepair/repairer/repairer.go

```diff
@@ -50,7 +50,7 @@ func (c Config) GetSegmentRepairer(ctx context.Context, tc transport.Client, met
 
 // SegmentRepairer is a repairer for segments
 type SegmentRepairer interface {
-	Repair(ctx context.Context, path storj.Path, lostPieces []int32) (err error)
```

Comment:
If we are removing `lostPieces` as an argument, we may as well remove it from the proto (`pb/datarepair.proto`)

Reply:
I agree, but on a second thought, it could be used for some metrics later on. Also I wanted to make small changes. Removing adds up few more file(s) changes. I just wanted to do incremental change.